README.md

```diff
@@ -1,70 +1,58 @@
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- CSS
+- React.js
+- Javascript
+- JSX
+- Babel
+- Redux
+- Linters
 
-## Available Scripts
+## Live Demo
+[Live Demo Link](Coming soon)
 
-In the project directory, you can run:
 
-### `npm start`
+Please get your browser update.
+To get a local copy up and running follow these simple steps.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Run the command below from the Terminal:
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+      git clone <URL> ' https://github.com/benardop/country-facts.git' in your terminal
 
-### `npm test`
+	  cd country-facts
+        run the following commend to install packages:
+        npm install
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Usage
+Run the following command in yout terminal to start your local server to run the App:
+npm run start
 
-### `npm run build`
+## Authors
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+👤 **Benard**
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Location: Kenya
+- GitHub: [benardop](https://github.com/benardop/)
+- Twitter: [@Bepacho](https://twitter.com/Bepacho)
+- LinkedIn: [Benard Pacho](https://www.linkedin.com/in/ochieng-benard-8264b815/)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## 🤝 Contributing
 
-### `npm run eject`
+Contributions, issues, and feature requests are welcome!
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+Feel free to check the [issues page](https://github.com/benardop/country-facts/issues).
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Show your support
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Give a ⭐ if you like this project!
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Acknowledgments
 
-## Learn More
+- Thanks to Microverse team and my session Coding Partners
 
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).